<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="hasilLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="grayButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Titik">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <addaction name="actionBrightness"/>
    <addaction name="actionSimple_Contrast"/>
    <addaction name="actionContrast_Stretching"/>
    <addaction name="actionNegative_Image"/>
    <addaction name="actionBiner_Image"/>
    <addaction name="actionHistogram_Gray"/>
    <addaction name="actionHistogram_RGB"/>
    <addaction name="actionHistogram_Equilization"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Geometri">
    <property name="title">
     <string>Operasi Geometri</string>
    </property>
    <widget class="QMenu" name="menuRotasi">
     <property name="title">
      <string>Rotasi</string>
     </property>
     <addaction name="actionMin45_deg"/>
     <addaction name="action45_deg"/>
     <addaction name="actionMin90_deg"/>
     <addaction name="action90_deg"/>
     <addaction name="action180_deg"/>
    </widget>
    <widget class="QMenu" name="menuResize">
     <property name="title">
      <string>Resize</string>
     </property>
     <addaction name="actionzoom_in"/>
     <addaction name="actionzoom_out"/>
     <addaction name="actionskew"/>
    </widget>
    <addaction name="actionTranslation"/>
    <addaction name="menuRotasi"/>
    <addaction name="menuResize"/>
    <addaction name="actionCrop_2"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Aritmatika">
    <property name="title">
     <string>Operasi Aritmatika</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionSubtract"/>
   </widget>
   <addaction name="menuOperasi_Titik"/>
   <addaction name="menuOperasi_Geometri"/>
   <addaction name="menuOperasi_Aritmatika"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Contrast</string>
   </property>
  </action>
  <action name="actionContrast_Stretching">
   <property name="text">
    <string>Contrast Stretching</string>
   </property>
  </action>
  <action name="actionNegative_Image">
   <property name="text">
    <string>Negative Image</string>
   </property>
  </action>
  <action name="actionBiner_Image">
   <property name="text">
    <string>Biner Image</string>
   </property>
  </action>
  <action name="actionHistogram_Gray">
   <property name="text">
    <string>Histogram Grayscale</string>
   </property>
  </action>
  <action name="actionHistogram_RGB">
   <property name="text">
    <string>Histogram RGB</string>
   </property>
  </action>
  <action name="actionHistogram_Equilization">
   <property name="text">
    <string>Histogram Equilization</string>
   </property>
  </action>
  <action name="actionOperasi_Geometri">
   <property name="text">
    <string>Operasi Geometri</string>
   </property>
  </action>
  <action name="actionTranslasi">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="actionTranslation">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="action_45_derajat">
   <property name="text">
    <string>-45 derajat</string>
   </property>
  </action>
  <action name="action45_derajat">
   <property name="text">
    <string>45 derajat</string>
   </property>
  </action>
  <action name="action_90_derajat">
   <property name="text">
    <string>-90 derajat</string>
   </property>
  </action>
  <action name="action90_derajat">
   <property name="text">
    <string>90 derajat</string>
   </property>
  </action>
  <action name="action180_derajat">
   <property name="text">
    <string>180 derajat</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom-in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom-out</string>
   </property>
  </action>
  <action name="actionSkew">
   <property name="text">
    <string>Skew</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionMin45_deg">
   <property name="text">
    <string>-45 derajat</string>
   </property>
  </action>
  <action name="action45_deg">
   <property name="text">
    <string>45 derajat</string>
   </property>
  </action>
  <action name="actionMin90_deg">
   <property name="text">
    <string>-90 derajat</string>
   </property>
  </action>
  <action name="action90_deg">
   <property name="text">
    <string>90 derajat</string>
   </property>
  </action>
  <action name="action180_deg">
   <property name="text">
    <string>180 derajat</string>
   </property>
  </action>
  <action name="actionCrop_2">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionzoom_in">
   <property name="text">
    <string>zoom-in</string>
   </property>
  </action>
  <action name="actionzoom_out">
   <property name="text">
    <string>zoom-out</string>
   </property>
  </action>
  <action name="actionskew">
   <property name="text">
    <string>skew</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="actionSubtract">
   <property name="text">
    <string>substract</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
